app/components/main/main.component.ts

```diff
@@ -83,6 +86,8 @@ export class MainComponent implements OnInit, MenuEventListener, REPLDelegate, T
 
     onContextChanged = () => { // arrow syntax to bind correct "this"
         if (this.context) {
+            this.context.graph.activeEdgeType = "DFAEdge";
```

Comment:
This is a simulation of the tools panel, so it has to be hard coded until that is built. It could be more realistic by getting the list of possible edge types and setting it to the first one.